idea_town/frontend/static-src/styles/modules/_sticky-header.scss

```diff
@@ -0,0 +1,7 @@
+.fixed-header {
+    position: fixed;
+    top: 0px;
+    left: 0px;
```

Comment:
Nit: change these 2 lines to just `0` (don't need units).
Bonus points for sorting them A-Z to make them easier to scan, and changing indent to 2 spaces to be consistent w/ _idea-view.scss above.